charts/haystack-api/pipelines/my_pipelines.yaml

```diff
@@ -0,0 +1,53 @@
+version: '1.1.0'
+
+components:    # define all the building-blocks for Pipeline
+  - name: DocumentStore
+    type: ElasticsearchDocumentStore
+    params:
+      host: localhost
+  - name: Retriever
+    type: ElasticsearchRetriever
+    params:
+      document_store: DocumentStore    # params can reference other components defined in the YAML
+      top_k: 5
+  - name: Reader       # custom-name for the component; helpful for visualization & debugging
+    type: FARMReader    # Haystack Class name for the component
+    params:
+      model_name_or_path: deepset/bert-large-uncased-whole-word-masking-squad2
+      context_window_size: 500
+      return_no_answer: true
+  - name: TextFileConverter
+    type: TextConverter
+  - name: PDFFileConverter
+    type: PDFToTextConverter
+  - name: Preprocessor
+    type: PreProcessor
+    params:
+      split_by: word
+      split_length: 1000
+  - name: FileTypeClassifier
+    type: FileTypeClassifier
+
+pipelines:
+  - name: query    # a sample extractive-qa Pipeline
+    type: Query
```

Comment:
Pipelines don't have a type anymore since haystack 1.2 I guess.

Reply:
@tstadel - Can I get an "default" example pipeline yaml? I am far from an expert when it comes to configuring pipelines in haystack.